src/data/translations/english.ts

```diff
@@ -80,7 +80,7 @@ const resources: LanguageFile = {
                 MESSAGE:
                     'A verification code has been sent to the email address you provided. Click the link or enter the code below to continue. This code is valid for 30 minutes.',
                 RESEND: 'Resend Verification Email',
-                VERIFICATION_CODE_PROMPT: "Didn't receive a code?",
+                VERIFICATION_CODE_PROMPT: "Didn't receive email?",
```

Comment:
```suggestion
                VERIFICATION_CODE_PROMPT: "Didn't receive an email?",
```

Reply:
Not sure how other languages handle the indefinite article, but may need to update the translations as well.

Reply:
I matched what was approved for NG workflow translations, but yes it should be Didn't receive an email? 
either way is fine for me.

Reply:
@daileytj  & @huayunh updated to include 'an' in english and other transalations